src/js/themes/base.d.ts

```diff
@@ -1025,7 +1026,8 @@ export interface ThemeType {
   };
   menu?: {
     background?: BackgroundType;
-    drop?: DropProps;
+    button?: ButtonType;
```

Comment:
I'm wondering if this should be called `item`, since the Menu is a DropButton already. It might align better with the `items` property.

Reply:
Good observation. That does make sense to me, I'll refine it.